### PR TITLE
Added new material

### PR DESCRIPTION
**Pull request by marcellosavoldi**
Added new material for the screws A2-70
see https://github.com/icub-tech-iit/cad-libraries/issues/74

### CODE REVIEW

File: config/materials/iso3506_a2-70.mtl [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  Access = Full
+},
+{
+  Name = PTC_SPECIFIC_HEAT
+  Type = Real
+  Default = 5.100000e+08 mm^2/(sec^2 C)
+  Access = Full
+},
+{
+  Name = PTC_TENSILE_YIELD_STRESS
```
Comment (fiorisi) [inline config/materials/iso3506_a2-70.mtl:104]: The yield should be 450 [MPa]: https://midlandbrightsteels.co.uk/portfolio/special-steels/a2-70/
I would add also the ultimate, that is actually 700 [MPa].

File: config/materials/iso3506_a2-70.mtl [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+{
+  Name = PTC_BEND_TABLE
+  Type = String
+  Default = ''
+  Access = Full
+},
+{
+  Name = PTC_DEFAULT_APPEARANCE
+  Type = String
+  Default = 'steel-stainless'
```
Comment (fiorisi) [inline config/materials/iso3506_a2-70.mtl:130]: Can you check if we have this appearance and how it looks like?

**Review by fiorisi — CHANGES_REQUESTED** [review]

File: config/materials/iso3506_a2-70.mtl (hunk 1 above)
Comment (marcellosavoldi) [reply config/materials/iso3506_a2-70.mtl:104]: @fiorisi , do you mean that:
image
?

File: config/materials/iso3506_a2-70.mtl [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
```
[the remark this answers is not in the document]
Comment (marcellosavoldi) [reply config/materials/iso3506_a2-70.mtl:31]: you're right!
what do you thgink about the folowing:
`DIN ES ISO 3506-1: 1997 for bolts, screws and nuts made of austenitic S.Steel`
... this is the maximum number of letters allowed!
@fiorisi

File: config/materials/iso3506_a2-70.mtl [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  Type = String
+  Default = 'Sample materials data from ANSYS Granta
+See grantadesign.com/PTC for more'
+  Access = Full
+},
+{
+  Name = GRANTA_DATALINK_VERSION
```
[the remark this answers is not in the document]
Comment (marcellosavoldi) [reply config/materials/iso3506_a2-70.mtl:38]: ok

File: config/materials/iso3506_a2-70.mtl (hunk 2 above)
Comment (marcellosavoldi) [reply config/materials/iso3506_a2-70.mtl:130]: I set this appearance by myself !
is the creo default for stainless steel
image

@fiorisi

File: config/materials/iso3506_a2-70.mtl (hunk 3 above)
[the remark this answers is not in the document]
Comment (fiorisi) [reply config/materials/iso3506_a2-70.mtl:31]: The technical standard is ISO 3506-1, we don't need to add the "DIN" or "ES".
Furthermore, the standard has been published in 2020: https://www.iso.org/standard/70045.html

**Review by fiorisi — APPROVED** [review]
Ok form me.

**Pull request merged**